app/build.gradle

```diff
@@ -243,9 +244,9 @@ dependencies {
         exclude group: "org.hisp.dhis", module: "core-rules"
     }
 
-    releaseImplementation("org.hisp.dhis.rules:rule-engine:${libraries.ruleEngine}") {
+  /*  releaseImplementation("org.hisp.dhis.rules:rule-engine:${libraries.ruleEngine}") {
         exclude group: "junit", module: "junit"
-    }
+    }*/
```

Comment:
This should not be uploaded, right?

Reply:
once we have the release version of the sdk it will revert back to default